Sources/SwiftDiscord/DiscordSnowflakeID.swift

```diff
@@ -72,6 +77,41 @@ public typealias IntegrationID = Snowflake
 /// A Snowflake ID representing an Attachment
 public typealias AttachmentID = Snowflake
 
+// MARK: Extra snowflake information
+
+extension Snowflake {
```

Comment:
I can't think of any uses for the three `Int` values, but the Snowflake <-> Timestamp are pretty useful.  Should I remove `workerId`, `processId`, and `numberInProcess`?